app/pb_kits/playbook/pb_button/docs/_button_full_width.md

```diff
@@ -0,0 +1 @@
+Used for mobile.
```

Comment:
You might say this is many times for mobile or other things like cards and sidebars. Buttons shouldn't stretch beyond 400px.